UI/frontend-plugins/frontend-tools/scripts.cpp

```diff
@@ -635,6 +623,36 @@ static void script_log(void *, obs_script_t *script, int log_level,
 				  Q_ARG(int, log_level), Q_ARG(QString, qmsg));
 }
 
+static const char *GetPythonPath()
+{
+	QString registryPath;
+	registryPath += "HKEY_CURRENT_USER\\Software\\Python\\PythonCore\\";
+	registryPath += PYTHON_VERSION;
+#if ARCH_BITS == 32
+	registryPath += "-32";
+#endif
+	registryPath += "\\InstallPath";
+
+	QSettings pySettings(registryPath, QSettings::NativeFormat);
+	QString path = pySettings.value(".").toString();
+
+	if (path.isEmpty()) {
+		registryPath = "";
+		registryPath +=
+			"HKEY_LOCAL_MACHINE\\SOFTWARE\\Python\\PythonCore";
```

Comment:
You're missing a `\\` on the end of this string to match the other path. Without it, it's read as
```
HKEY_LOCAL_MACHINE\SOFTWARE\Python\PythonCore3.6\InstallPath
```
which is incorrect. This'd be why it doesn't work for @Scrxtchy. Adding the `\\` certainly fixed it for me.

Reply:
Done.